### PR TITLE
Update dependencies.props

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -5,22 +5,22 @@
 
   <PropertyGroup Label="Package Versions">
     <CastleCorePackageVersion>4.2.1</CastleCorePackageVersion>
-    <InternalAspNetCoreSdkPackageVersion>3.0.0-build-20181204.1</InternalAspNetCoreSdkPackageVersion>
+    <InternalAspNetCoreSdkPackageVersion>3.0.0-build-20181114.5</InternalAspNetCoreSdkPackageVersion>
```
Comment (natemcmaster) [inline build/dependencies.props]: Can you leave this version as is?

Comment (sebastienros) [reply build/dependencies.props]: Done

**Comment by sebastienros** [comment]
The windows build is failing very early now.

**Comment by natemcmaster** [comment]
The build error appeared to be unrelated to your change. I've clicked "retry" for you. 

(For future reference, you should be able to do this, too. When signed into Azure Pipelines, you should see a 'Retry' button if you click the triple dots on the top right)

image

**Comment by sebastienros** [comment]
I did that, I think you can see it in the history. Thanks anyway.

**Pull request merged**